Libs/Mesh/Mesh.cpp

```diff
@@ -53,8 +54,6 @@
 #include <vtkAppendPolyData.h>
 #include <vtkCleanPolyData.h>
 #include <vtkNew.h>
-#include <vtkPolyData.h>
```

Comment:
You'll wish you hadn't removed these when you've consolidated VtkMeshWrapper. Out of curiosity, what informed you they were no longer necessary?